assets/style/style.css

```diff
@@ -407,41 +425,92 @@ body {
   gap: 15px;
 }
 
+.project__popup .works__item {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  gap: 15px;
+}
+
 .tonic__details {
   display: flex;
   flex-direction: column;
   gap: 15px;
 }
 
-.header__container {
+.header__container,
+.header__container2 {
   height: 100%;
   width: 100%;
   border-radius: 8px;
   overflow: hidden;
 }
 
-.header__container div {
-  height: 100%;
-  width: 100%;
-  background-size: cover !important;
-  border-radius: 8px;
-  transition: 1s ease-in;
+.see__project_popup {
+  display: none;
+  transition: all 0.5s ease;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 99;
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
 }
 
-.tonic__header div {
-  background: no-repeat center url(../img/card_tonic.svg);
+.act {
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
-.tonic2__header div {
-  background: no-repeat center url(../img/card_tonic2.svg);
+.project__popup {
+  width: 90%;
+  height: 90%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  overflow: scroll;
+  padding: 1rem;
```

Comment:
- Please, let's use the correct color for the paragraph in the popup and in the `ABOUT SECTION` to make the design close to the Figma as much as we can. See image below:

*Figma design*
![Screenshot from 2023-03-08 14-06-48](https://user-images.githubusercontent.com/59397159/223722714-aa6bab09-b136-48e0-9239-4497be232436.png)

*Your design*
![Screenshot from 2023-03-08 14-05-21](https://user-images.githubusercontent.com/59397159/223722720-d57b9c81-8f29-4e02-95f7-f6cf560b993d.png)

*Figma design*
![Screenshot from 2023-03-08 14-23-05](https://user-images.githubusercontent.com/59397159/223724580-268ca922-8b85-436e-baf7-fd62cfd86ea7.png)

*Your design*
![Screenshot from 2023-03-08 14-22-52](https://user-images.githubusercontent.com/59397159/223724593-2f9cf725-a37a-4d60-9c9c-51b0b0fdc7da.png)